wps_tools/utils.py

```diff
@@ -12,9 +12,80 @@
 import logging
 import os
 from pathlib import Path
+from urllib.parse import urlparse
+from urllib.request import urlretrieve
+from collections import OrderedDict
 
 MAX_OCCURS = 1000
 
+common_status_percentages = {
+    "start": 0,
+    "process": 20,
+    "build_output": 95,
+    "complete": 100,
+}
```

Comment:
Thank you for adding this in here. There are a lot of processes that would save code with this.